docs/CODE_GUIDELINES.md

```diff
@@ -280,82 +415,200 @@ enum Dummy
 };
 ```
 
-### 11.4. Interfaces
-Use CamelCase for interface names and they have to be prefixed with an uppercase I. Filename has to match the interface name without the prefixed I, e.g. Logger.h
+### 8.4. Interfaces
+Use PascalCase and prefix with an uppercase I. Filename has to match the interface name without the prefixed I, e.g. Logger.h
 ```cpp
 class ILogger
 {
-  void Log(...) = 0;
+public:
+  virtual void Log(...) = 0;
+  virtual ~ILogger() {}
```

Comment:
You should mention that overriding methods shouldn't use `virtual`, only `override`.

Reply:
Yeah I even forgot about that in one example :-D